benchmarks/src/jmh/java/com/linecorp/armeria/common/stream/StreamMessageBenchmark.java

```diff
@@ -128,11 +128,12 @@ private void writeNextValue(StreamWriter<Integer> stream) {
     }
 
     @Benchmark
-    public long noExecutor(StreamObjects streamObjects) {
+    public long noExecutor(StreamObjects streamObjects) throws InterruptedException {
         final StreamMessage<Integer> stream = newStream(streamObjects);
         stream.subscribe(streamObjects.subscriber);
         streamObjects.writeAllValues(stream);
-        // No executor, so sum will be updated inline.
+        // ensure the event loop closes the stream before checking the computed sum
+        streamObjects.completedLatch.await(10, TimeUnit.SECONDS);
```

Comment:
How about adding `stream.whenComplete().join()` to the first line of `computedSum()` though it produces a warn message. 
```
private long computedSum() {
    stream.whenComplete().join();
    final long computedSum = subscriber.sum();
    ...
}
```